### PR TITLE
Add some tools to improve DX when working on castor

### PR DESCRIPTION
**Pull request by pyrech**

### CODE REVIEW

File: tools/php-cs-fixer/castor/run.php [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace cs;
```
Comment (lyrixx) [inline tools/php-cs-fixer/castor/run.php:3]:
```suggestion
namespace qa;
```

File: tools/php-cs-fixer/castor/run.php
Comment (lyrixx) [inline tools/php-cs-fixer/castor/run.php:1]: I love the idea to put the castor file in this repository 💛

File: tools/phpstan/castor/run.php [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace phpstan;
```
Comment (lyrixx) [inline tools/phpstan/castor/run.php:3]:
```suggestion
namespace qa;
```

**Review by lyrixx — COMMENTED** [review]
Nice!

File: tools/phpstan/castor/run.php [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+use Castor\Attribute\AsTask;
+
+use function Castor\run;
+
+#[AsTask(description: 'Run PHPStan', aliases: ['phpstan'])]
+function phpstan(): void
+{
+    run([__DIR__ . '/../vendor/bin/phpstan', '--configuration=' . \dirname(__DIR__, 3) . '/phpstan.neon'], allowFailure: true);
```
Comment (lyrixx) [inline tools/phpstan/castor/run.php:12]: Why do you allow failure?

File: tools/php-cs-fixer/castor/run.php [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+
+use Castor\Attribute\AsTask;
+
+use function Castor\run;
+
+#[AsTask(description: 'Fix CS', aliases: ['cs'])]
+function cs(): void
+{
+    run([__DIR__ . '/../vendor/bin/php-cs-fixer', 'fix', '--config', \dirname(__DIR__, 3) . '/.php-cs-fixer.php'], allowFailure: true);
```
Comment (lyrixx) [inline tools/php-cs-fixer/castor/run.php:12]: same here

**Comment by pyrech** [comment]
Updated

**Review by lyrixx — APPROVED** [review]
Nice ! 

So now you can use theses tools in the CI?

**Comment by lyrixx** [comment]
Thanks

**Pull request merged**